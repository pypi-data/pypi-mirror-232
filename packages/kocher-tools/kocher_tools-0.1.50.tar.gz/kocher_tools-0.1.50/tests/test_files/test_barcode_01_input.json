[
    {
        "Query ID": "DBtest-A2_1;size=500",
        "Status": "Ambiguous Hits",
        "Species": [
            "Ceratina mikmaqi",
            "Ceratina strenua"
        ],
        "Bins": [
            "BOLD:AAA2368"
        ]
    }
]
