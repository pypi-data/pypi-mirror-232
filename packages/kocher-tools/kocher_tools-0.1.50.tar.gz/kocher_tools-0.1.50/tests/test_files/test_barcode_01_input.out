Query ID	Query Length	Subject ID	Subject Length	Percent Identity	Alignment Length	Mismatches	Gaps	Query Alignment Start	Query Alignment End	Subject Alignment Start	Subject Alignment End	E-Value	Bitscore
DBtest-A1_1;size=500	313	LASIO003-06|Lasioglossum_oenotherae|BOLD:AAD9268|COI-5P	658	100.000	313	0	0	1	313	346	658	1.92e-164	579
